manim/animation/creation.py

```diff
@@ -153,8 +155,42 @@ def _get_bounds(self, alpha: float) -> typing.Tuple[int, float]:
         return (0, alpha)
 
 
-class Uncreate(ShowCreation):
-    """Like :class:`ShowCreation` but in reverse.
+class ShowCreation(ShowPartial):
+    """Deprecated class. Use Create() instead.
+
+        Incrementally show a VMobject.
+    Parameters
+    ----------
+    mobject : :class:`~.VMobject`
+        The VMobject to animate.
+    Raises
+    ------
+    :class:`TypeError`
+        If ``mobject`` is not an instance of :class:`~.VMobject`.
+    Examples
+    --------
+    .. manim:: ShowCreationScene
+        class ShowCreationScene(Scene):
+            def construct(self):
+                self.play(ShowCreation(Square()))
+    See Also
+    --------
+    :class:`~.ShowPassingFlash`
+    """
+
+    logger.warning(
+        "ShowCreation has been deprecated in favor of Create. Please use Create instead!"
+    )
+
+    def __init__(self, mobject: VMobject, lag_ratio: float = 1.0, **kwargs) -> None:
+        super().__init__(mobject, lag_ratio=lag_ratio, **kwargs)
+
+    def _get_bounds(self, alpha: float) -> typing.Tuple[int, float]:
+        return (0, alpha)
```

Comment:
I'd suggest to handle the actual deprecation differently, like outlined in https://stackoverflow.com/a/53450052.

Essentially, this would mean that this should only be something like

```py
class ShowCreation(Create):
    """Deprecated. Use :class:`~.Create` instead."""
    def __init_subclass__(self):
        logger.warning(...)
```

In particular, I wouldn't duplicate the docstring for the deprecated class.

Reply:
Oh, makes sense to omit the redundant docstring indeed. Thank you!

Reply:
Also, be sure to change ShowCreation to inherit from Create instead of ShowPartial